http-client/src/main/java/io/airlift/http/client/jetty/DefaultHttpClientLogger.java

```diff
@@ -67,6 +68,10 @@
             rollingPolicy.setFileNamePattern(rollingPolicy.getFileNamePattern() + ".gz");
         }
 
+        // Limit total log files occupancy on disk. Ideally we would keep exactly
```

Comment:
Also update `rollingPolicy.setMaxHistory(maxHistory);` line above, with `// limits number of logging periods (i.e. days) kept` comment

Reply:
got it